packages/instrumentation-aws-sdk/test/dynamodb.spec.ts

```diff
@@ -69,4 +66,57 @@ describe('DynamoDB', () => {
             });
         });
     });
+
+    describe('BatchGetIem', () => {
+        beforeEach(() => {
+            mockAwsSend(responseMockSuccess, {
+                Responses: { 'test-table': [{ key1: { S: 'val1' } }] },
+                UnprocessedKeys: {},
+                ConsumedCapacity: [
+                    {
+                        TableName: 'test-table',
+                        CapacityUnits: 0.5,
+                        Table: { CapacityUnits: 0.5 },
+                    },
+                ],
+            } as AWS.DynamoDB.Types.BatchGetItemOutput);
+            instrumentation.disable();
+            instrumentation.enable();
```

Comment:
@blumamir Hey! Just curious, do you know why I need to `disable` and `enable` the instrumentation every time before we run a test? I noticed the tests won't have spans (even though it will have a response) if we don't do this?

Reply:
Actually, it's always been kind of voodoo to me as well. It's a practice copied from the instrumentations in contrib which sometimes helped tests pass in setups where we have multiple `.spec` files.

Recently, I investigated the issue deeply and was able to trace the problems to multiple instances of instrumentation class living together in the same mocha process which causes a mess in patching the module functions, sometimes causing tests to run with no patching enabled thus with no spans as well. 
I guess that `enable` and `disable` were somehow triggering fresh patching of the module before each suite started.

My current approach is not to call `enable` and `disable`, but to have a single instance of the instrumentation class implemented as a singleton [here](https://github.com/aspecto-io/opentelemetry-ext-js/blob/master/packages/instrumentation-testing-utils/src/instrumentation-singelton.ts).
This practice has been used in amqplib instrumentation, and I plan to apply it to the tests in other instrumentations as well. [This PR is where I first introduced the new singleton approach](https://github.com/aspecto-io/opentelemetry-ext-js/pull/159)

I can fix this issue for aws-sdk on Sunday if you experience pain in writing tests for this PR.

Reply:
Ah okay thanks for the background! Seems like a very viable cause 😄 

Thanks for the offer! 🙂 Should be no need right at this moment, I got the tests to pass on this PR.

Not in a rush but please let me know what you think about this PR now that it is ready! This is not urgent. It's just a small PR  to instrument `DynamoDB.BatchGetItem` and includes a test based on the response I saw in my local test. Thank you! Have a good weekend! 😄

Reply:
Sure, I'll review it at the beginning of next week.

Thank you again for contributing and doing such a thorough job.

Have a great weekend :)